test/validators.js

```diff
@@ -6054,18 +6054,19 @@ describe('Validators', () => {
       {
         locale: 'nl-NL',
         valid: [
-          '0670123456',
```

Comment:
I'm a little concerned about the regression here? You mean the previous one was completely off?

Reply:
This particular line would succeed. 0670123456 would be correct. But it was allowed to omit the 6 previously. That is not correct for a dutch mobile number. The 6 is not optional. 

Actually the rules are quite simple. Its always a 10 digit number starting with 06. Therefore I replaced the numbers in the testcase with more telling examples (counting 1 to 8)
The only exception is when the country code is included. This, in most cases will replace the 0 in 06 for +31 or verbosely 0031. Alternatively the 0 in 06 may be added between braces when preceded by the country code. Eg: +31612345678, 0031612345678, +31(0)612345678

Reply:
Got it. For the previous test cases, could you strike out only those that do not pass the tests now. I really want to be careful about regressions right now, and not clearly whenever we have any breaking changes.

Reply:
I will readd these testcases to the valid list
'0670123456',
'+31670123456',
'31670123456',

and add the incorrect ones to the invalid list
'021234567',
'+3121234567',
'3121234567',

Reply:
of course implementations of this might fail, because if consumers expected the validator to be right, then they might have added incorrect numbers already.

Reply:
so the regression for consumers will require some updates on their side